bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,276 @@
+import logging
+import textwrap
+from collections import ChainMap
+from typing import Union
+
+from aiohttp.client_exceptions import ClientResponseError
+from discord import Color, Embed, Member, User
+from discord.ext.commands import Context, group
+
+from bot.constants import Channels, Guild, Roles, Webhooks
+from bot.decorators import with_role
+from bot.pagination import LinePaginator
+from .watchchannel import WatchChannel, proxy_user
+
+log = logging.getLogger(__name__)
+STAFF_ROLES = Roles.owner, Roles.admin, Roles.moderator, Roles.helpers    # <- In constants after the merge?
```

Comment:
Yes, I'll take care of this when merging. It was probably the most frequent conflict I dealt with in the merge.

Reply:
Okay, I'll leave it as it is for now, then.